homeassistant/components/blue_current/entity.py

```diff
@@ -0,0 +1,63 @@
+"""Entity representing a Blue Current charge point."""
+from homeassistant.const import ATTR_NAME
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from . import Connector
+from .const import DOMAIN, MODEL_TYPE
+
+
+class BlueCurrentEntity(Entity):
+    """Define a base Blue Current entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, connector: Connector, signal: str) -> None:
+        """Initialize the entity."""
+        self.connector: Connector = connector
```

Comment:
The `connector` parameter is already typed. We don't need to type it again here. Same for the other attributes below.